topology/src/main/java/io/camunda/zeebe/topology/metrics/TopologyMetrics.java

```diff
@@ -55,6 +61,20 @@ public final class TopologyMetrics {
           .name("cluster_changes_operations_completed")
           .help("Number of completed changes in the current change plan")
           .register();
+  private static final Gauge OPERATION_DURATION =
+      Gauge.build()
+          .namespace(NAMESPACE)
+          .name("cluster_changes_operation_duration")
+          .help("Duration it takes to apply an operation")
+          .labelNames(LABEL_OPERATION)
+          .register();
+  private static final Counter OPERATION_ATTEMPTS =
+      Counter.build()
+          .namespace(NAMESPACE)
+          .name("cluster_changes_operation_attempts")
+          .help("Number of retries per operation type")
```

Comment:
:thinking: This not number of retries, right? It is more like "Number of attempts (failed or applied) per operation type"